benchmarks/Exporter/JaegerExporterBenchmarks.cs

```diff
@@ -89,7 +89,7 @@ public async Task JaegerExporter_Batching()
             {
                 for (int c = 0; c < this.NumberOfSpans; c++)
                 {
-                    await jaegerUdpBatcher.AppendAsync(this.testSpan, CancellationToken.None).ConfigureAwait(false);
+                    await jaegerUdpBatcher.AppendAsync(this.testSpan.ToJaegerSpan(), CancellationToken.None).ConfigureAwait(false);
```

Comment:
Prior to this PR `ToJaegerSpan()` was called inside `AppendAsync` so the amount of work in the benchmark it is still the same.